modules/standard/Random.chpl

```diff
@@ -504,15 +583,92 @@ module Random {
 
 
     /*
-      Models a stream of pseudorandom numbers.  See the module-level
-      notes for :mod:`PCGRandom` for details on the PRNG used.
+
+      Models a stream of pseudorandom numbers generated by the PCG random number
+      generator.  See http://www.pcg-random.org/ and the paper, "PCG: A Family
+      of Simple Fast Space-Efficient Statistically Good Algorithms for Random
+      Number Generation" by M.E. O'Neill.
+
+      This class builds upon the :record:`pcg_setseq_64_xsh_rr_32_rng` PCG RNG
+      which has 64 bits of state and 32 bits of output.
+
+      While the PCG RNG used here is believed to have good statistical
+      properties, it is not suitable for cryptographic use. Additionally,
+      if statistical properties of the random numbers are very important,
+      another strategy may be required.
+
+      We have verified that the random numbers generated by this class can match
```

Comment:
"can match" sure is waffle-y.  I understand why (we haven't checked all streams for arbitrary lengths), but it still gives less confidence than seems ideal.  "We have good confidence that the random numbers ... match"?